opm/simulators/aquifers/BlackoilAquiferModel_impl.hpp

```diff
@@ -25,6 +25,9 @@ template <typename TypeTag>
 BlackoilAquiferModel<TypeTag>::BlackoilAquiferModel(Simulator& simulator)
     : simulator_(simulator)
 {
+    // Grid needs to support Facetag
+    assert(supportsFaceTag(simulator.vanguard().grid()));
```

Comment:
static_assert - we want this error on compile time.

Reply:
static_assert gives 

`non-constant condition for static assertion`

Reply:
I am awfully sorry for suggesting something, that I have never tried. Seems we ran into a [gcc bug](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=77446). 

See my old-fashioned solution in PR totto82/opm-simulators#10